modules/ServerAPI/src/client/Protocol.html

```diff
@@ -112,7 +112,23 @@
 
         </div>
     </div>
-
+    <div class="bv_endpointManagerSection">
+        <h3>Endpoints</h3>
+        
+        <div class="bv_strictEndpointMatchingCheckbox">
+            <input class= "bv_strictEndpointMatchingInputCheckbox" type="checkbox" id="strictEndpointMatchingCheckbox" name="strictEndpointMatchingCheckbox">
+            <span style="display: inline-block; position: relative; bottom: -3px;">Strict Endpoint Matching</span>
+            <br/>
+            <br/>
+            <p><i>When Strict Endpoint Matching is enabled, experiment uploads that try to use columns not defined below will be rejected with an error. When this option is disabled, ACAS will simply give warnings about these columns but allow them to proceed, and they'll appear in the list below after upload.</i></p>
+            <br/>
+            <p class="bv_endpointManagerInstructions"><i>Click on a row to view experiments using protocols with that data.</i></p>
```

Comment:
There are configs "client.protocol.label" and client.experiment.label. The intent provide some uniformity around nomenclature between live design and ACAS.

 ACAS "Protocols" get mapped to Live Design "Assays" and ACAS "Experiments" to Live Design "Assay Protocols"

Id like to try and keep that up as much as we can.  Can we have this message and other messages follow that pattern?  If you search for `protocol.label` or `experiment.label` you can find a bunch of ways it is using to match lower case or plural forms. This will apply for message coming from R as well.